### DISCUSSION TITLE
Runewords

### DISCUSSION THREAD

**Discussion by spitfire2112** [isid0re/SoloLeveling#166] [Q&A] [answered]
I'd like to have the bot make and keep the runeword enigma. I think I've written this correctly but I wanted someone to verify for me. Will adding the following two lines make this work correctly?

Config.Runewords.push([Runeword.Enigma, "Wyrmhide", Roll.NonEth]); 
Config.KeepRunewords.push("[type] == armor # [plusskillteleport] == 1");

Thank you

**Comment by isid0re** [comment, marked as answer]
What class are you using? Most of the classes already make and equip enigma.  And if you need a working example you can take it from one of the class.sololeveling.js config files.

**Reply by spitfire2112** [reply]
I was unaware that it would make and equip that already, and I've used it on most of the working builds. I probably haven't let it run long enough to farm the items. 
Thank you for the quick reply.

**Reply by isid0re** [reply]
No problem. And thanks for reaching out and using the bot.
